src/font.c

```diff
@@ -159,6 +161,13 @@ GdipPrivateAddFontFile (GpFontCollection *font_collection,  GDIPCONST WCHAR *fil
 	if (!file)
 		return OutOfMemory;
 
+	fileHandle = fopen ((char *)file, "r");
```

Comment:
`GdipPrivateAddFontFile` returns `FileNotFound` if the file doesn't exist with GDI+. Previously libgdiplus returned `Ok`